react/features/prejoin/functions.js

```diff
@@ -126,6 +171,79 @@ export function getDeviceStatusType(state: Object): string {
     return state['features/prejoin']?.deviceStatusType;
 }
 
+/**
+ * Returns the full dial out number (containing country code and +).
+ *
+ * @param {Object} state - The state of the app.
+ * @returns {string}
+ */
+export function getFullDialOutNumber(state: Object): string {
```

Comment:
Please alpha order the functions too. I know we have some files too where they are not (we're actively fighting against it, but we fail occasionally)

Reply:
Fixed.